libmamba/CMakeLists.txt

```diff
@@ -526,7 +526,7 @@ macro(libmamba_create_target target_name linkage output_name)
 
             if(APPLE)
                 set(MAMBA_FORCE_DYNAMIC_LIBS resolv c++abi)
-                target_link_options(${target_name} PRIVATE -static-libstdc++)
+                target_link_options(${target_name} PRIVATE -nolibstdc++)
             elseif(UNIX)
                 set(MAMBA_FORCE_DYNAMIC_LIBS rt dl resolv)
                 target_link_options(${target_name} PUBLIC -static-libstdc++ -static-libgcc)
```

Comment:
```suggestion
                target_link_options(${target_name} PUBLIC -nolibstdc++ -static-libgcc)
```

Reply:
The `-static-libstdc++` works on other UNIX systems, it's broken on APPLE only.